app/controllers/solidus_square/base_controller.rb

```diff
@@ -9,6 +9,10 @@ class BaseController < ::Spree::BaseController
 
     private
 
+    def order_token
```

Comment:
This method is only required by the API controllers.
What do you think to create a specific API BaseController where move it there, and let the API controllers inherit from it?

Reply:
https://github.com/nebulab/solidus_square/pull/49/commits/591b5191d9fd9f444e56482bf3019ac8c5393bfb